projects/ngx-editor/src/lib/icons/index.ts

```diff
@@ -30,36 +36,40 @@ const icons: Record<string, any> = {
   code,
   underline,
   strike,
-  ordered_list: orderedList,
-  bullet_list: bulletList,
+  ordered_list,
+  bullet_list,
   blockquote: quote,
   link,
   unlink,
   image,
-  align_left: alignLeft,
-  align_center: alignCenter,
-  align_right: alignRight,
-  align_justify: alignJustify,
-  text_color: textColor,
-  color_fill: colorFill,
-  horizontal_rule: horizontalRule,
-  format_clear: formatClear,
+  align_left,
+  align_center,
+  align_right,
+  align_justify,
+  text_color,
+  color_fill,
+  horizontal_rule,
+  format_clear,
+  path,
 };
 
 class Icon {
   static get(name: keyof typeof icons, fill = DEFAULT_ICON_FILL): string {
-    const path = icons[name] || '<path></path>';
-    return `
-      <svg
-        xmlns="http://www.w3.org/2000/svg"
-        viewBox="0 0 24 24"
-        fill=${fill}
-        height=${DEFAULT_ICON_HEIGHT}
-        width=${DEFAULT_ICON_WIDTH}
-      >
-        ${path}
-      </svg>
-    `;
+    const fullPath = icons[name];
+    if (fullPath.includes('<path') || fullPath.includes('<g')) {
```

Comment:
will this help when providing icons over config? I understand it, but not sure the entire idea behind the logic.

Reply:
If you add icons as image or class like `<i class='fa fa-bold'></i>` you dont need interpolate the svg, and for that I check if I need or not the interopolation.